code/datums/ammo/ammo.dm

```diff
@@ -236,16 +239,17 @@
 		var/final_angle = initial_angle
 
 		var/obj/projectile/P = new /obj/projectile(curloc, original_P.weapon_cause_data)
-		P.generate_bullet(GLOB.ammo_list[bonus_projectiles_type]) //No bonus damage or anything.
+		P.generate_bullet(GLOB.ammo_list[bonus_projectiles_type])
+		P.damage *= gun_damage_mult
```

Comment:
You should probably just copy the other bullets damage mult instead. similar to how accuracy is calculated on the next line